.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
This file is added for debugging a yo template using vscode

Reply:
So what scenario gets enabled using this?

Reply:
When a developer is making some changes in the yo template then he can use vsode to debug if he faces any issues. He needs to enter the process id of the yo process and select debug option from vscode. By doing this debugger is attached to the yo process and he can proceed for debugging

Reply:
Thanks.. Can we document this on the readme so that people know how to use this? Also should we add this in all yo repos?

Reply:
Yes will do that

Reply:
Updated the readme accordingly.